packages/arb-token-bridge-ui/src/hooks/useNetworksAndSigners.tsx

```diff
@@ -356,10 +350,7 @@ export function NetworksAndSignersProvider(
 
     const fallbackProps =
       result.status === UseNetworksAndSignersStatus.NOT_SUPPORTED
-        ? {
-            status: result.status,
-            chainId: result.chainId
-          }
+        ? { status: result.status }
         : { status: result.status }
```

Comment:
no need for the ternary, just return the same object regardless

Reply:
Next PR will remove this file entirely, so I didn't want to complicate the diff